src/test/java/com/amazon/ion/impl/bin/IonManagedWriter_1_1_Test.kt

```diff
@@ -709,24 +711,25 @@ internal class IonManagedWriter_1_1_Test {
         // However, this should be held loosely.
         val expected = """
             $ion_1_1
-            $ion_encoding::(
-              (symbol_table ["foo","bar","baz"])
-              (macro_table
+            (:$ion::set_symbols
+              (:: "foo" "bar" "baz"))
```

Comment:
It would be nice if we could have it write it like this instead, but we can leave that for another day.
```
(:set_symbols "foo" "bar" "baz")
```

Reply:
I looked into that briefly- in some cases it is being written that way, so it looks like there's some more thought needed on the tuning or something else I missed.